internal/util/validation/validation.go

```diff
@@ -94,6 +94,47 @@ func ValidateString(s *string, path string, minLen int, maxLen int, patternRegex
 	return asErrors(errs)
 }
 
+func ValidateCronField(field string, min, max int, fieldName string) error {
+	if field == "*" {
+		return nil
+	}
+
+	// break down the field into its parts
+	parts := strings.Split(field, ",")
+	for _, part := range parts {
+		if strings.Contains(part, "-") { // Validate ranges
+			rangeParts := strings.Split(part, "-")
+			if len(rangeParts) != 2 {
+				return fmt.Errorf("invalid range in %s field: %q", fieldName, field)
```

Comment:
Could it be better to quote the fieldName
return fmt.Errorf("invalid range in %q field: %s", fieldName, field)

Reply:
yeah copilot malfunction here :)